flask_admin/contrib/sqla/filters.py

```diff
@@ -118,6 +120,38 @@ def operation(self):
         return lazy_gettext('not in list')
 
 
+# Composite filters
+class FilterAndEqual(FilterEqual):
+    def __init__(self, child_filters, name, delimiter=',', options=None, data_type=None):
+        column = [filter_.column for filter_ in child_filters]
+        super(FilterAndEqual, self).__init__(column, name, options, data_type)
+        self.filters = child_filters
+        self.delimiter = delimiter
+
+    def convert_aliases(self, aliases):
+        is_aliases_sequence = isinstance(aliases, collections.Sequence) and not isinstance(aliases, text_type)
+        aliases = [aliases] * len(self.filters) if not is_aliases_sequence else aliases
+        return aliases
+
+    def apply(self, query, values, aliases=None):
+        aliases = self.convert_aliases(aliases)
+        for filter_, value, alias in zip(self.filters, values, aliases):
+            query = filter_.apply(query, value, alias)
+        return query
+
+    def clean(self, value):
+        result = value.split(self.delimiter)
+        if len(result) != len(self.filters):
+            names = [filter_.name for filter_ in self.filters]
+            value_format = self.delimiter.join(names)
+            raise ValueError('Value format should be: {value_format}.'.format(value_format=value_format))
```

Comment:
This message doesn't make it to the UI. It's caught by `validate` and it says something like this instead `Invalid Filter Value: test`. This is an important message too, because there's nothing else indicating how many items are required.